src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment } from '../Comment/Comment';
+
+export const CommentList = ({ comments }) => (
+  <ul className="comments">
+    {comments.map(comment => (
+      <li className="comment" key={comment.id}>
+        <Comment {...comment} />
```

Comment:
давай краще писати так
```suggestion
        <Comment comment={comment} />
```